elastic_search.go

```diff
@@ -856,12 +856,12 @@ func createWorldResults(ctx context.Context, user *users.User, tx *gorm.DB, elas
 		}
 	}
 
-	return worldsProto, count
+	return &worldsProto, count
```

Comment:
This was changed because we're returning a proto data type; proto data types contain a `sync.Mutex` that `go vet` suggests we avoid passing as values. Using pointers solves this issue.

```
Return copies the lock value: type 'fuel.Worlds' contains 'protoimpl.MessageState' contains 'sync.Mutex' which is 'sync.Locker' 
```